### PR TITLE
New package: wsdd-0.6.4

### PR DESCRIPTION
**Pull request by Mek101**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

I requested wsdd myself some time ago: ended up doing it myself https://github.com/void-linux/void-packages/issues/28216

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me

### CODE REVIEW

**Comment by Mek101** [comment]
Added samba as a dependency since it's useless without a samba server on the host

**Comment by Mek101** [comment]
The build fail in x86_64-musl is caused by python and unrelated to this package (however, I'm on x84_64-musl, commit 53c249b, and it works)
Waiting for the ok to squash the commits.

**Comment by Mek101** [comment]
Squashed

File: srcpkgs/wsdd/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'wsdd'
+pkgname=wsdd
+version=0.6.4
+revision=1
+archs="*"
```
Comment (ericonr) [inline srcpkgs/wsdd/template:5]: You can simply omit the `archs` variable.

**Review by ericonr — COMMENTED** [review]
@ahesford would appreciate a quick look.

File: srcpkgs/wsdd/template (hunk 1 above)
Comment (Mek101) [reply srcpkgs/wsdd/template]: Removed!

**Comment by ahesford** [comment]
Seems to work as advertised. To test, I
1. Launched the program as `wsdd -s` as a regular user.
2. Opened "Network" on a Win10 machine on my network; my Void server was visible.
3. Killed `wsdd`; the Void server disappeared from the Win10 "Network" view.
4. Restarted `wsdd -s`; after a few seconds, the server reappeared.

**Comment by Mek101** [comment]
Bump

**Pull request closed without merging**